pkg/generate/rbac/parser_test.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbac
+
+import (
+	"go/token"
+	"reflect"
+	"testing"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+)
+
+func TestParseFile(t *testing.T) {
+	tests := []struct {
+		content string
+		exp     []rbacv1.PolicyRule
+	}{
+		{
+			content: `package foo
+	import (
+		"fmt"
+		"time"
+	)
+
+	// RBAC annotation with kubebuilder prefix
+	// +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;delete
```

Comment:
We may want to have more test cases to cover
1) core group
2) using `""`

Reply:
Added.